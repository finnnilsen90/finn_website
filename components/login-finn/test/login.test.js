const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')
let { shallow, mount, render } = require('enzyme');

let Background = require("../../../images/santa_cruz2015.jpg");

const Login = require('../login-finn.js');
const Box = require('../lib/box-login/box-login.js')

let LoginConst = new Login(); 

jest.mock('../lib/box-login/box-login.js', () => 'Box')

describe('component connection', () => {

    it('should be defined', () => {
        expect(Login).toBeDefined();
    })

    it('img should be defined', () => {
        expect(Background).toBeDefined();
    })

})  
   
describe('<Login />', () => {

    const wrapper = render(<Login />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });


})
