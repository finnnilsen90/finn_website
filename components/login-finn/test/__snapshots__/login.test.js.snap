// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> renders components 1`] = `
<div
  class="containter"
>
  <img
    class="background_img"
    src="test-file-stub"
  />
  <div
    class="box_cont"
  >
    <box />
  </div>
</div>
`;
